docs/earthly-command/earthly-command.md

```diff
@@ -324,22 +337,127 @@ These options can only be set via environment variables, and have no command lin
 | EARTHLY_TARGET_PADDING | `EARTHLY_TARGET_PADDING=n` will set the column to the width of `n` characters. If a name is longer than `n`, its path will be truncated and remaining extra length will cause the column to go ragged. |
 | EARTHLY_FULL_TARGET    | `EARTHLY_FULL_TARGET=1` will always print the full target name, and leave the target name column ragged.                                                                                                   |
 
-## earthly prune
+
+## earthly --version
```

Comment:
Moved earthly --version, earthly ls, earthly doc to the top of the doc as it felt awkward to have them halfway down